gen/dcos-config.yaml

```diff
@@ -235,11 +235,26 @@ package:
                 "name": "com_mesosphere_mesos_JournaldLogger",
                 "parameters": [
                   {
+                    "key": "destination_type",
+                    "value": "{{ mesos_container_log_destination }}"
+                  }, {
                     "key": "companion_dir",
                     "value": "/opt/mesosphere/active/mesos-modules/bin/"
                   }, {
                     "key": "libprocess_num_worker_threads",
                     "value": "2"
+                  }, {
+                    "key": "max_stdout_size",
+                    "value": "2MB"
+                  }, {
+                    "key": "logrotate_stdout_options",
+                    "value": "rotate 9"
+                  }, {
+                    "key": "max_stderr_size",
+                    "value": "2MB"
+                  }, {
+                    "key": "logrotate_stderr_options",
+                    "value": "rotate 9"
```

Comment:
In addition to these module options from the previous logrotate module, what about `launcher_dir`? Is that superseded by `companion_dir`?